emperor/support_files/js/emperor.js

```diff
@@ -2261,25 +2274,23 @@ $(document).ready(function() {
 		}
 		document.getElementById("labels").innerHTML = labelshtml;
 
-		labelshtml = "";
-		// add the labels with the taxonomic lineages to the taxalabels div
-		for(var key in g_taxaPositions){
-
-			// get the coordinate of this taxa sphere
-			var coords = toScreenXY(g_plotTaxa[key].position,g_sceneCamera,$('#main-plot'));
-
-			// labels are identified by the key they have in g_taxaPositions
-			labelshtml += "<label id=\""+key+"_taxalabel\" class=\"unselectable labels\" style=\"position:absolute; left:"+parseInt(coords['x'])+"px; top:"+parseInt(coords['y'])+"px;\">";
-			labelshtml += g_taxaPositions[key]['lineage'];
-			labelshtml += "</label>";
-		}
-		document.getElementById("taxalabels").innerHTML = labelshtml
-
+                displayTaxaLabels(0);
+	        //add sliding bar to specify taxonomic labeling
+                $("#selectTaxaLevel").slider(
+                {
+                            value:0,
+                            min: 0,
+                            max: 7,
```

Comment:
This number should only go up to the highest level that we have available in the taxa strings.